src/dpp/cluster/emoji.cpp

```diff
@@ -42,4 +42,44 @@ void cluster::guild_emojis_get(snowflake guild_id, command_completion_event_t ca
 	rest_request_list<emoji>(this, API_PATH "/guilds", std::to_string(guild_id), "emojis", m_get, "", callback);
 }
 
+	//me.id.str()
+
+void cluster::application_emojis_get(command_completion_event_t callback) {
+	/* Because Discord can't be consistent, we can't just do `rest_request_list<emoji>` because all items are behind `items`.
+	 * so now we end up with this duplicating `rest_request_list` because we need to iterate the `items` array!!! Thanks Discord!!!!!!!
```

Comment:
perhaps!!!!!! not so many!!!!! exclamation marks!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 🤣

Reply:
🤣 I'll maybe tone them down!!!!!!!!!